src/common/Utilities/Util.h

```diff
@@ -21,6 +21,7 @@
 
 #include "Define.h"
 #include "Errors.h"
+#include "Common.h"
```

Comment:
Will inline still be handled properly if definition is in the cpp? Don't want to introduce extra call overhead, and the likelyhood of something using Util.h without using Common.h is practically nonexistant.